### PR TITLE
feat: dashboard widget panel fit and finish 2306

### PR DESCRIPTION
**Pull request by Priyanshu44**
image

Issue: https://github.com/awslabs/iot-app-kit/issues/2036

Description: Added "Widgets" text with horizontal divider to the Widget library panel on the dashboard.

Changes in the CR:
-> Added "Widgets" header with grey-900 font color and a right divider of 2px.
->Added bottom border, 2px grey-200

### CODE REVIEW

**Review by mnischay — DISMISSED** [review]

**Review by tracy-french — DISMISSED** [review]

**Comment by tracy-french** [comment]
One of the status checks is stuck and there's a message at the top about "Processing updates". Will check in later to check if it can be merged.

File: packages/react-components/src/components/chart/utils/getXAxis.ts [hunk 1]
```diff
@@ -19,6 +19,6 @@ export function getXAxis(axis?: ChartAxisOptions): XAXisOption {
     },
     splitNumber: 6,
     min: 0,
-    max: Date.now(),
+    max: viewportInMs.isDurationViewport ? viewportInMs.end : undefined,
```
Comment (corteggiano) [inline packages/react-components/src/components/chart/utils/getXAxis.ts:22]: does this break gesturing? specifically scrolling to the left and right on the charts

File: .github/workflows/codeql-analysis.yml [hunk 2]
```diff
@@ -13,9 +13,9 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, rc ]
```
Comment (mnischay) [inline .github/workflows/codeql-analysis.yml:16]: these changes are already in the rc branch ?

**Review by mnischay — CHANGES_REQUESTED** [review]
something is messed with the CR, it has unrealted changes

**Comment by Priyanshu44** [comment]
Resolved conflict

**Comment by Priyanshu44** [comment]
Resolved conflict

**Pull request closed without merging**